wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -47,6 +47,7 @@ <h1 class="header__title">
                     </div>
                 {% endif %}
             {% endblock %}
+            <!-- START:END -->
```

Comment:
Suggestion: seeing this comment out of context in a template file might be confusing. Could we add an explanatory django comment above?